compiler-rt/lib/asan/asan_internal.h

```diff
@@ -130,6 +130,19 @@ void InstallAtExitCheckLeaks();
   if (&__asan_on_error) \
   __asan_on_error()
 
+// Depending on the loading thread and when ASAN is loaded on Windows,
+// race conditions can appear causing incorrect states or internal check
+// failures.
```

Comment:
Is it still true with changes from that patch?

Also, it would be nice, if explanation from that comment would be in a PR description and commit message.

Reply:
Apologies, I'm trying to fulfill the repo's PR requirements here: https://github.com/llvm/llvm-project/pull/71833#issue-1986121343

This is just one of the spr PRs created. It appears I've done it incorrectly, though. I'll be closing this PR.